### PR TITLE
Add initial pom file and lint checks to project

### PR DESCRIPTION
**Pull request by acheng1**
You can also run these commands locally

`make pretty to format code`
`make validate to validate the code`

### CODE REVIEW

**Comment by acheng1** [comment]
@googlebot I signed it!

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <plugins>
+      <!-- Provides `mvn package appengine:run` for local testing
+           and `mvn package appengine:deploy` for deploying. -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+          <!-- TODO: set project ID. -->
+          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```
Comment (bryyeh) [inline pom.xml:38]: Do you need to put the project id here?

Comment (acheng1) [reply pom.xml:38]: I did that but didn't push that change. Pushed now.

**Comment by acheng1** [comment]
Moved to https://github.com/googleinterns/step12-2020/pull/4

**Pull request closed without merging**